New/add query sharing docs

tonilastre opened the pull request:
### Description

Add all new environment variables for the Lab. Add query-sharing feature page.

For the Lab release: Lab 2.15.

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [x] New documentation page, release related

### Related PRs and issues

- 

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [x] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

kgolubic reviewed (approved):
I've made some updates to styling so that Vercel build can pass. I've also made some minor language changes. Everything looks good to me.

hal-eisen-MG commented on pages/data-visualization/install-and-connect.mdx:
```diff
@@ -196,24 +197,67 @@ The desktop application version of Memgraph Lab does not support receiving envir
+| `AUTH_SAML_OKTA_ISSUER`                  | Okta SAML issuer                                                        | `string`   |                    |
+| `AUTH_SAML_OKTA_SIGNATURE_ALGORITHM`     | Okta SAML signature algorithm                                           | `string`   | `"sha256"`         |
+| `ENTERPRISE_LICENSE_ORG_NAME`            | Enterprise license organization name                                    | `string`   |                    |
+| `ENTERPRISE_LICENSE_KEY`                 | Enterprise license key                                                  | `string`   |                    |
```
Mention something about "contact our Sales team via XYZ to purchase a key"

tonilastre replied:
Actually, I am thinking about adding a link for both ENV variables to https://memgraph.com/docs/database-management/enabling-memgraph-enterprise - it seems better to have one page that describes the enterprise rather than to have "copy" of that description in multiple places. The enterprise link also has a link to request a license or trial.

hal-eisen-MG commented on pages/data-visualization/install-and-connect.mdx:
```diff
@@ -196,24 +197,67 @@ The desktop application version of Memgraph Lab does not support receiving envir
+| `AUTH_SAML_OKTA_CALLBACK_URL`            | Okta SAML callback URL                                                  | `string`   |                    |
+| `AUTH_SAML_OKTA_ISSUER`                  | Okta SAML issuer                                                        | `string`   |                    |
+| `AUTH_SAML_OKTA_SIGNATURE_ALGORITHM`     | Okta SAML signature algorithm                                           | `string`   | `"sha256"`         |
+| `ENTERPRISE_LICENSE_ORG_NAME`            | Enterprise license organization name                                    | `string`   |                    |
```
Explain the significance of this variable

tonilastre replied:
Currently, I added a link to the enterprise license documentation. Maybe it makes sense to add everything about the license there to have it in one place.

This is the only link I've found:  https://memgraph.com/docs/database-management/enabling-memgraph-enterprise

hal-eisen-MG commented on pages/data-visualization/user-manual/query-sharing.mdx:
```diff
@@ -0,0 +1,110 @@
+will be able to handle their own query shares.
+
+If there are no users in the Memgraph database, then anyone connecting
+to Memgraph acts as the same user (not defined one) so everyone can
```
s/not defined one/Not-Defined user

hal-eisen-MG commented on pages/data-visualization/user-manual/query-sharing.mdx:
```diff
@@ -0,0 +1,110 @@
+When a viewer clicks on the query share link, the viewer will need
+to connect to Memgraph first and will be presented with the query
+to run. This way, any sensitive information within your query or
+parameters are not shared publicly but contained within the Lab and
```
s/but contained/but rather are safely contained

hal-eisen-MG commented on pages/data-visualization/user-manual/query-sharing.mdx:
```diff
@@ -0,0 +1,110 @@
+---
```
Is it possible to LIST all queries which have been stored for sharing?

Is it possible to EDIT an existing shared query?

Please address both questions in the doc. If "no", then explain about possible future expansion of the feature.

tonilastre commented:
I've updated Query sharing docs and added all the reported fixes.

Note: In the query sharing docs I've added `TODO` parts where images and/or video should be added, some are sketches, and some are screenshots. Please consult with Ana and the team for them to have them ready by Wed.
